BUILDING.md

```diff
@@ -13,3 +13,17 @@ Beyond that page, note:
 
 Building with autoconf is deprecated.  While there are many --with
 options, the build system mostly follows autoconf norms.
+
+# Building from vcpkg
+
+The gdal port in vcpkg is kept up to date by Microsoft team members and community contributors. The url of vcpkg is: https://github.com/Microsoft/vcpkg . You can download and install gdal using the vcpkg dependency manager:
```

Comment:
The below instructions are to get GDAL binaries from the latest version supported by the vcpkg recipee, whereas this page is more about to create a GDAL development environment.
The section you've added should more be moved to https://gdal.org/download.html#binaries (there's an "Edit on github" button at the top right of the page to edit the corresponding .rst file)

Reply:
Thanks for your reply. I have created a new PR https://github.com/OSGeo/gdal/pull/6302 to submit these change to **download.rst**.